.github/workflows/build_linux_jax_wheels.yml

```diff
@@ -126,3 +126,21 @@ jobs:
           source .venv/bin/activate
           pip3 install boto3 packaging
           python3 ./build_tools/third_party/s3_management/manage.py ${{ inputs.s3_subdir }}/${{ inputs.amdgpu_family }}
+
+  build_test_push_jax_docker:
```

Comment:
How come we didn't use the `.github/workflows/test_jax_dockerfile.yml` workflow that was already there?